AwesomeProject/App.js

```diff
@@ -24,8 +25,10 @@ const styles = StyleSheet.create({
   header: {
```

Comment:
Styles should be in styles.js file. You should create AboutMe.styles.js file in your case ,where you export this styles.

Reply:
Fixed